templates/agents/holidays_hours.html.twig

```diff
@@ -44,11 +44,11 @@ $(document).ready(function() {
       <td>Nombre d'heures de congés par an :</td>
       <td style='text-align:right;'>
         {% if can_manage_agent %}
-          <input type='text' id='conges_annuel' name='conges_annuel' value='{{ annuel_heures}}'  style='width:70px;text-align:right;'>
+          <input type='text' id='conges_annuel_hours' name='conges_annuel_hours' value='{{ annuel_heures }}'  style='width:70px;text-align:right;'>
           <label style='text-align:center;padding:5px;'>h</label>
-          {% include 'agents/elements/select_holiday_minutes.html.twig' with {'minutes': annuel_min, 'name': 'conges_annuel_min' } %}
+          <input type='text' name='conges_annuel_min' id='conges_annuel_min' value='{{ annuel_min | digit(2) }}' style='width:70px;text-align:left;'>
         {% else %}
-          {{ annuel_string }}
```

Comment:
* annuel_string et $annuelString ne sont plus utilisé avec Conges-Mode = Heures : à nettoyer
* $annuelString semble être utilisé avec Conges-Mode = Jours : à vérifier

Reply:
Les variables $xxxString sont nécessaires à la conversion des heures en jours lorsque cette option est activée.
Ces variables étant définie avec une chaîne vide et non alimentée ensuite, la conversion ne fonctionne plus et affiche systématiquement "0 jours"

A corriger

Doc :
https://docky.biblibre.com/procedures/Proc%C3%A9dures/Parametrages_specifiques_configuration_Planno.html#mt29618-add-hours-to-days-custom-conversion

Reply:
> * annuel_string et $annuelString ne sont plus utilisé avec Conges-Mode = Heures : à nettoyer

Oui, c'est le cas: le commit 358567396197efffbc665416cb38488934808c06 a supprimé annuel_string de templates/agents/holidays_hours.html.twig et $annuelString de la partie Conges-Mode = Heures dans src/Controller/AgentController.php

> * $annuelString semble être utilisé avec Conges-Mode = Jours : à vérifier

Oui, il est utilisé Conges-Mode = Jours

Reply:
> Les variables $xxxString sont nécessaires à la conversion des heures en jours lorsque cette option est activée. Ces variables étant définie avec une chaîne vide et non alimentée ensuite, la conversion ne fonctionne plus et affiche systématiquement "0 jours"
> 

Corrigé par 74bd42831ca883749abd554a39d9f84ce6ceb112

Reply:
revue et tests : OK